pkg/driver/controller.go

```diff
@@ -52,13 +53,14 @@ const (
 	volumeParamsDailyAutomaticBackupStartTime = "dailyAutomaticBackupStartTime"
 	volumeParamsDeploymentType                = "deploymentType"
 	volumeParamsDiskIopsConfiguration         = "diskIopsConfiguration"
+	volumeParamsOptionsOnDeletion             = "optionsOnDeletion"
```

Comment:
mega nit: It's organized filesystem, filesystem delete, volume, volume delete, and then alphabetical so this would go above SkipFinalBackup